backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/ProprietaryController.java

```diff
@@ -104,4 +105,32 @@ public ResponseEntity<ProprietarySchema> updateProprietaryAtDate(
       return ResponseEntity.notFound().build();
     }
   }
+
+  /**
+   * Return specific metadata of a single element at a given date within the {@link Proprietary}
+   * node.
+   *
+   * @param eli Eli of the ZF0 version of the target law where the metadata are saved.
+   * @param eid the eId of the single element within the ZF0
+   * @param atDate the time boundary at which to return the metadata
+   * @return the specific metadata returned in the form of {@link ProprietarySingleElementSchema}
+   */
+  @GetMapping(
+      path = "/{eid}/{atDate}",
+      produces = {APPLICATION_JSON_VALUE})
+  public ResponseEntity<ProprietarySingleElementSchema> getProprietaryAtDate(
+      final Eli eli, @PathVariable final String eid, @PathVariable final LocalDate atDate) {
+
+    try {
+      var proprietary =
+          loadProprietaryFromNormUseCase.loadProprietaryFromNorm(
+              new LoadProprietaryFromNormUseCase.Query(eli.getValue()));
+
+      return ResponseEntity.ok(
+          ProprietaryResponseMapper.fromProprietarySingleElement(proprietary, eid, atDate));
+
+    } catch (NormNotFoundException e) {
```

Comment:
We will soon merge the branch with the exception handling in the backend and we won't need to catch the NormNotFoundException as it will be handled in the controller advice. Sebastian will remove the try/catching of the old endpoints so I would say why not also remove it here. I don't think we would have problems till Sebastian's branch is merged or do we have an integration controller test that is testing the not found exception and what http code is returned?

Reply:
Yes, there is.